services/src/test/scala/cromwell/services/database/LobSpec.scala

```diff
@@ -35,7 +35,7 @@ class LobSpec extends AnyFlatSpec with CromwellTimeoutSpec with Matchers with Sc
       containerOpt.foreach { _.start }
     }
 
-    it should "fail to store and retrieve empty blobs" taggedAs DbmsTest in {
+    it should "store empty blobs" taggedAs DbmsTest in {
```

Comment:
Was this just always incorrectly named?

Reply:
Not "always". At some point the body of the test was updated and the name of the test wasn't.